src/main/resources/static/css/app.css

```diff
@@ -1,143 +0,0 @@
-body {
```

Comment:
Since the UI looks fine, then apparently I don't know what I'm missing :) other than the mod drop area. That took some CSS-fu to get laid out right.